scripts/githooks/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
You can already give a list of paths to format.bash; I put that in at your request : )